lib/ui/process_ui/widgets/radio_button_control.dart

```diff
@@ -49,7 +49,7 @@ class _RadioFormFieldState extends State<RadioButtonControl> {
   void _getSelectedValueFromMap(String lang) {
     String response = "";
     response =
-        context.read<GlobalProvider>().fieldInputValue[widget.field.id][lang];
+        context.read<GlobalProvider>().fieldInputValue[widget.field.id];
```

Comment:
why is this option removed?

Reply:
Aviral changes was also added in patch-fixes this is that only

Reply:
let's cherry pick changes from here https://github.com/mosip/android-registration-client/pull/279/files